packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx

```diff
@@ -162,9 +162,8 @@ class AudioPlayerWithYoutubeSpotify extends Component {
       audioSource = {};
       audioSource[channelToPlay] = albumSource;
     }
-    const trackToHighlight = trackSelected === null && trackStartingPoint !== null ? trackStartingPoint : trackSelected;
-    audioSource = find(tracklistToShow, track => track.trackNumber === trackToHighlight);
-
+    // If selected track not null find info for selcted track, else find for first track of list of selected channel
+    audioSource = trackSelected ? tracklistToShow.find(t => t.trackNumber === trackSelected) : tracklistToShow.find(t => t.trackNumber >= 1);
```

Comment:
I'm curious what happens when a CD is presented with a track number of 0. This album, for example, ripped the media before the first track marker as "Hidden track one audio". With this change wouldn't the player always skip that 0th track?

https://archive.org/details/cd_burn_sister-machine-gun

Reply:
In this case track 0 is actually assigned the trackNumber 1 (and we're only depending on trackNumber to decide the track to be loaded)

track 0
![Screen Shot 2019-07-19 at 3 06 00 PM](https://user-images.githubusercontent.com/30471843/61525869-e9dc5e00-aa36-11e9-9b8c-bcb61465fd49.png)

track 1
![Screen Shot 2019-07-19 at 3 06 10 PM](https://user-images.githubusercontent.com/30471843/61525870-ea74f480-aa36-11e9-97a4-e000808ed0ee.png)

Reply:
👍